android/app/src/main/AndroidManifest.xml

```diff
@@ -61,6 +61,8 @@
             <data android:scheme="https" android:host="getraha-alternate.app.link" />
             <data android:scheme="https" android:host="getraha.test-app.link" />
             <data android:scheme="https" android:host="getraha-alternate.test-app.link" />
+            <data android:scheme="https" android:host="link.raha.app" />
```

Comment:
Is this what gets sent to user? Does branch require that we use `link` instead of something shorter?

Reply:
Yes, it's what gets sent to the user. No, we can do something shorter. Suggestions that isn't "d"? We settled on "link" since it's pretty short, descriptive, and l is an ambiguous character

Reply:
We're gonna do TO.raha.app!!! because it takes you TO the Raha App wowwww